packages/server/modules/workspaces/repositories/workspaceJoinRequests.ts

```diff
@@ -8,6 +11,12 @@ const tables = {
     db<WorkspaceJoinRequest>(WorkspaceJoinRequests.name)
 }
 
+export const createWorkspaceJoinRequestFactory =
+  ({ db }: { db: Knex }): CreateWorkspaceJoinRequest =>
+  async (request) => {
+    return await tables.workspaceJoinRequests(db).insert(request)
```

Comment:
Does this really fit the `CreateWorkspaceJoinRequest` type? Unless you specify `.insert(request, '*')` its not gonna return the created request. And even with that it would return it in an array